vllm/transformers_utils/config.py

```diff
@@ -568,3 +570,12 @@ def try_get_generation_config(
             return GenerationConfig.from_model_config(config)
         except OSError:  # Not found
             return None
+
+
+def get_cross_encoder_activation_function(config: PretrainedConfig):
+    if (hasattr(config, "sbert_ce_default_activation_function")
+            and config.sbert_ce_default_activation_function is not None):
+        return import_from_string(
+            config.sbert_ce_default_activation_function)()
```

Comment:
For better security, perhaps we should limit the modules that can be imported this way? Though I guess it may break compatibility with sentence-transformers...